caluma/settings.py

```diff
@@ -28,8 +28,10 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 # Application definition
 
 INSTALLED_APPS = [
-    "django.contrib.contenttypes",
     "django.contrib.postgres",
+    "localized_fields",
+    "psqlextra",
```

Comment:
Is the `psqlextra` app a dependency of `localized_fields`? Or why do we need this?

Reply:
yes localized_fields requires psqlextra (it add extra feature to HStoreField)